pkg/k8s/apis/cilium.io/v2/register.go

```diff
@@ -396,6 +396,10 @@ func createNodeCRD(clientset apiextensionsclient.Interface) error {
 											Description: "min-allocate is the minimum number of IPs that will be allocated before" +
 												" the cilium-agent will write the CNI config.",
 										},
+										"max-allocate": {
+											Type:        "integer",
+											Description: "max-allocate is the maximum number of IPs that will be allocated to the node.",
+										},
 										"pre-allocate": {
```

Comment:
Once you are touching these, would you mind adding a description to `pre-allocate` please?

Reply:
sure! I have also done it for `max-above-watermark` as well 👌